reframe/frontend/statistics.py

```diff
@@ -205,7 +205,7 @@ def print_failure_report(self, printer):
         run_report = self.json()[-1]
         last_run = run_report['runid']
         for r in run_report['testcases']:
-            if r['result'] == 'success' or r['result'] == 'aborted':
+            if r['result'] == 'success' or r['result'] == 'aborted' or r['result'] == 'skipped':
```

Comment:
```suggestion
            if r['result'] in {'success', 'aborted', 'skipped'}:
```

Reply:
Agreed, that's more elegant and compact. I just updated the PR :)